scripts/PalletTown.asm

```diff
@@ -271,30 +271,45 @@ PalletTownText_19002:
 	text_far _OakWhewText
 	text_end
 
-PalletTownText8: ; girl
+PalletTownText8:
 	text_far _OakGrassText
 	text_end
 
-PalletTownText2: ; fat man
+PalletTownText2: ; girl
 	text_far _PalletTownText2
 	text_end
 
-PalletTownText3: ; sign by lab
+PalletTownText3: ; fat man
 	text_far _PalletTownText3
 	text_end
 
-PalletTownText4: ; sign by fence
+PalletTownText4: ; sign by lab
 	text_far _PalletTownText4
 	text_end
 
-PalletTownText5: ; sign by Red's house
+PalletTownText5: ; sign by fence
+IF DEF(_DEBUG)
+	text_asm
+	ld a, 239
```

Comment:
How about `ld a, $ef ; ???` because this makes no sense.

Reply:
The sign post in Pallet Town prints `240bit` so I think `239` is a more reasonable representation.